### ISSUE TITLE
Worms Blast (70650)

### ISSUE THREAD

**Issue by iodream** [ValveSoftware/Proton#706]
The game launches but has heavy visual glitching in the menus where in the character selection menu half of the screen is just black. And it crashes as soon as the game match begins.
Had both of these problems with latest wine-staging prior to this if it helps with anything.
It also has a hard time after being minimized/alt-tabbed, as most games with SteamPlay for now. Results in a black screen.
I'm using Proton 3.7-4 beta on Ubuntu 18.04.1 Bionic and tested with Mesa 18.0.5

[SteamPlay log](https://gist.github.com/iodream/04685009cf345bebf3b8fea6a3cb32b4)

[PC specs](https://gist.github.com/iodream/ad63408e3749f9137dc2c3fef1cbbe2b)

Thanks a lot for any help.

**Comment by Zero86Sk** [comment]
I had a black backround, with the menu items and when going into a stage it would freeze, i had to go to a terminal and kill the process.

My Specs:
https://gist.github.com/Zero86Sk/d629f657c1ae861c36657cdf238a7765

**Comment by grabmateusz** [comment]
## System Information
- GPU: GeForce GT 540M
- Driver/LLVM version: NVidia driver 390.48
- Kernel version: 4.15.0-33-generic
- [Link to full system information report](https://gist.github.com/grabmateusz/9df08b1e0ec5ab3cf81fa4b8431c3bca)
- Proton version: 3.7-6 Beta

## I confirm:
- [x] that I have checked whether there are updates for my system available.

[Game log for session with black screen](https://gist.github.com/grabmateusz/726845933adb66fd9fde24fa24f10bc1)

## Symptoms
Exacltly like @iodream described - in menu there are some graphical glitches (while picking a character), after picking a level game crashes.

## Reproduction
Run a game, pick character, pick a level.

**Comment by joshas** [comment]
Still same issues on Proton 6.3-7 and current Proton Experimental: black or missing backgrounds, crash after picking level.

![worms-blast-menu](https://user-images.githubusercontent.com/79141/142038781-867e6f8a-ae82-41de-a541-097da7e2e145.png)

**Comment by iodream** [comment]
Upstream vanilla Wine seem to have fixed the graphical glitches and the crashing issues as per [Wine bug report](https://bugs.winehq.org/show_bug.cgi?id=54898) and the [MR](https://gitlab.winehq.org/wine/wine/-/merge_requests/2821)  in Wine 8.9.  I've tested the game launching through Wine steam and it works. And after using `winetricks directmusic` there is even music playing

But Proton 8.0 and Proton Experimental just crash the game after title screen.  [Proton log](https://github.com/ValveSoftware/Proton/files/13230017/steam-70650.log) 

Why is this happening? Is this because of extra Proton patches on top of Wine? Thanks

**Comment by iodream** [comment]
Latest Proton 1699285506 experimental-bleeding-edge-8.0-62216-20231106-pa090d1-w78c3b5-d2ed177-vf85d94

is back to the originally described behavior of glitching menus and crash after level select

[Proton log](https://github.com/ValveSoftware/Proton/files/13272091/steam-70650.log)

**Comment by iodream** [comment]
As of Proton 9(Beta) the original glitches are no longer present. The game still has some issues like no music and texture bugs with player characters though, so i'm not sure if this issue should be closed or not.

**Comment by iodream** [comment]
Retested with Proton 11.0-2. Game launches and works but there is still no menu music or in level background music. Most likely the same bug with the grey character model textures remains. 

There is also another bug: with Proton 10 there is now a blue outline inside the fullscreen window on the left and top sides of the screen. This is not present with Proton 9.

**Comment by codeweaverwill** [comment]
The blue outline seems to be fixed in Proton 11 Experimental on my end, and I'm not seeing texture issues there either. No directmusic fix yet, though.

@iodream let me know if you're still seeing the blue outline using Proton Experimental.

**Comment by iodream** [comment]
You're right the blue outline and character texture issues seem to both be gone. Retested with Proton 1788287817 experimental-bleeding-edge-11.0-425930-20260901-pdb08aa-w445631-dadeda6-v18c088-x86_64.

The music is still not present . 

Thank you very much for the update.
